pydatastructs/linear_data_structures/tests/test_arrays.py

```diff
@@ -21,6 +21,35 @@ def test_OneDimensionalArray():
     assert raises(TypeError, lambda: ODA(int, set([1, 2, 3])))
     assert raises(ValueError, lambda: ODA(int, 3, [1]))
 
+
+def test_MultiDimensionalArray():
+    array = MultiDimensionalArray(int, 5, 9, 3, 8)
+    assert array.shape == (5, 9, 3, 8)
```

Comment:
This isn't working as expected. `array.shape` is giving, `(2560, 576, 24, 8, 1)`. See, https://travis-ci.com/github/codezonediitj/pydatastructs/builds/164219241 for details.

Reply:
@JeanPierreMR Can you please look into the issue?

Reply:
Yes, sure
the idea of shape is that it saves the size of each slice of array.
In this case the 2560 is the full size of the array, 576 its the size of the slices of the first dimension.... 8 is the size of the second to last dimension, and 1 is the last one
the point of having it like this is to avoid calculating it every time you want to access something. 


I could:

- make 2 variables to save both, one called shape and one sizes (I think this would be the best)
- calculate the (5, 9, 3, 8), from the (2560, 576, 24, 8, 1) to access it
- calculate the size (2560, 576, 24, 8, 1) every time you want to access something

Reply:
Earlier we weren't saving the `shape` of the array and only `sizes` were being used? So, may be shape is just needed when `.shape` is called. We can go with, the second approach.
In addition, you can rename, `_shape` to `_sizes` as I got your point of using it.